### DISCUSSION TITLE
Personal question

### DISCUSSION THREAD

**Discussion by xamelion** [community/community#8288] [Copilot Conversations] [answered]
Good day to all. Sorry that the issue is a little off topic. I am programming with the typescript language and have been in a waiting mode for confirmation for four months now. I constantly look at youtube how copilot works and really want to try it too =) If possible, please give me access.

**Comment by davecheney** [comment, marked as answer]
Im sorry you’ve had to wait so long. We’re adding new invites as we have capacity. Thank you for your continued patience and understanding.
